src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,19 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private static final int FIGURE_COUNT = 100;
+    private Random random = new Random();
+    private ColorSupplier colSup = new ColorSupplier();
+
+    public Ball getRandomBall() {
+
+        Ball ball = new Ball();
+        ball.setColor(colSup.getRandomColor().name());
+        ball.setNumber(random.nextInt(FIGURE_COUNT));
+
+        return ball;
+
+    }
```

Comment:
[Please don’t add redundant empty lines to your code.](https://mate-academy.github.io/jv-program-common-mistakes/java-core/oop/lottery#please-dont-add-redundant-empty-lines-to-your-code)